pkg/cmd/util/output/backup_describer.go

```diff
@@ -895,6 +898,10 @@ func (v *volumesByPod) Add(namespace, name, volume, phase string, progress veler
 	// append backup progress percentage if backup is in progress
 	if phase == "In Progress" && progress.TotalBytes != 0 {
 		volume = fmt.Sprintf("%s (%.2f%%)", volume, float64(progress.BytesDone)/float64(progress.TotalBytes)*100)
+	} else if phase == string(velerov1api.PodVolumeBackupPhaseCompleted) && incrementalBytes > 0 {
+		volume = fmt.Sprintf("%s (size: %v, incremental size: %v)", volume, progress.TotalBytes, incrementalBytes)
+	} else if phase == string(velerov1api.PodVolumeBackupPhaseCompleted) && progress.TotalBytes > 0 {
```

Comment:
This `else if` will cause the `totalBytes` not being printed when `incrementalBytes > 0`, which is not correct, we should print both.